libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -2243,6 +2256,8 @@ const char *AP_Mission::Mission_Command::type() const
         return "Parachute";
     case MAV_CMD_DO_SPRAYER:
         return "Sprayer";
+    case MAV_CMD_DO_AUX_FUNCTION:
+        return "AuxSwitch";
```

Comment:
Should this be AuxFunction instead of AuxSwitch?

Reply:
Fixed.